apps/README.md

```diff
@@ -0,0 +1,37 @@
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development. It uses the [react-native-test app](https://github.com/microsoft/react-native-test-app) under the covers, and loads the FluentUI bundle.
+
```

Comment:
nit: loads the fluent-tester bundle, not FluentUI bundle

Reply:
Fixed in recent commit.